mock/py/mockbuild/config.py

```diff
@@ -324,7 +324,11 @@ def setup_default_config_opts():
     config_opts['dnf5_disable_plugins'] = []
     # No --allowerasing with remove, per
     # https://github.com/rpm-software-management/dnf5/issues/729
-    config_opts["dnf5_avoid_opts"] = {"remove": ["--allowerasing"]}
+    config_opts["dnf5_avoid_opts"] = {
+        "remove": ["--allowerasing"],
+        "groupinstall": ["--allowerasing"],
+        "group": ["--allowerasing"],
```

Comment:
`group install` will support --allowerasing in the next release https://github.com/rpm-software-management/dnf5/pull/1174